mod_reforged/hooks/config/strings.nut

```diff
@@ -1454,12 +1461,12 @@ foreach (vanillaDesc in vanillaDescriptions)
 	}),
 	RF_Ghostlike = ::UPD.getDescription({
 		Fluff = "Blink and you\'ll miss me.",
-		Requirement = "Total armor weight up to 20"
+		Requirement = "Total armor weight of 20 or less"
```

Comment:
Longer requirement strings look uglier in the tooltip so `up to 20` is better imo.

Reply:
up to 20 isn't really proper though imo. It's saying you're required to have armor weight, which isn't true. We could do `Total armor weight below 21`? I saw it done this way in one other place.

Reply:
I think the meaning of `up to 20` is identical to that of `20 or less`. Because it means you can't exceed 20. It doesn't mean you are required to have any.